corehq/apps/geospatial/forms.py

```diff
@@ -96,6 +96,7 @@ class Meta:
         help_text=_("The travel mode of the users. "
                     "Consider this when specifying the max travel time to each case."),
         widget=Select(choices=GeoConfig.VALID_TRAVEL_MODES),
+        validators=[validate_travel_mode]
```

Comment:
:+1:

Reply:
oh actually, what is this? In forms I believe the recommended way is to use `clean_` methods.
On models, we have validators.

Reply:
@mkangia I followed [this example](https://docs.djangoproject.com/en/5.0/ref/validators/#:~:text=Because%20values%20are%20converted%20to%20Python%20before%20validators%20are%20run%2C%20you%20can%20even%20use%20the%20same%20validator%20with%20forms%3A) from the django docs. But maybe if you think I should rather move it to the clean methods I can do that?

Reply:
(maybe because I _can_ doesn't mean I _should_)

Reply:
Cool. I might be limited by what I know here then. But this is better than calling assert.